.github/workflows/update_php_repo.yml

```diff
@@ -0,0 +1,45 @@
+name: Update protobuf-php Repo
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+
```

Comment:
I think this needs to be expanded to include RCs. I also suspect the "."  should also be escaped.

Reply:
Added a new tag to match on RCs.